### PR TITLE
[MM-60837] Replace FormattedMarkdownMessage in 'webapp/channels/src/components/user_settings/security/user_access_token_section/user_access_token_section.tsx' with FormattedMessage

### LINKED ISSUE #28597: Replace FormattedMarkdownMessage in 'webapp/channels/src/components/user_settings/security/user_access_token_section/user_access_token_section.tsx' with FormattedMessage
The goal of this task is to remove all instances of “FormattedMarkdownMessage” (imported from “components/formatted_markdown_message”) in the file `webapp/channels/src/components/user_settings/security/user_access_token_section/user_access_token_section.tsx`. These should be replaced with the “FormattedMessage” component from the “react-intl” library.

Ensure that rich text formatting is preserved by following the implementation guidelines in the react-intl (https://formatjs.io/docs/react-intl/components#rich-text-formatting) documentation. Make sure to change the message ID after the update as the current extraction tool does not automatically detect changes in the messages. Once done, make sure to run `npm run i18n-extract` in the “webapp/channels” directory to pick up the new messages. For guidance refer to the example pull request - https://github.com/mattermost/mattermost/pull/28438

----
If you're interested please comment here and come [join our "Contributors" community channel](https://community.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team. For technical advice or questions, please  [join our "Developers" community channel](https://community.mattermost.com/core/channels/developers).

New contributors please see our [Developer's Guide](https://developers.mattermost.com/contribute/getting-started/).

JIRA: https://mattermost.atlassian.net/browse/MM-60837

### PR DESCRIPTION
**Pull request by ZubairImtiaz3**
#### Summary
This PR replaces all instances of FormattedMarkdownMessage with FormattedMessage from the react-intl library in user_access_token_section.tsx, ensuring rich text formatting is preserved. The message IDs have been updated, and npm run i18n-extract was executed to capture the new messages.

#### Ticket Link
Fixes #28597
Jira: [MM-60837](https://mattermost.atlassian.net/browse/MM-60837)

#### Release Note

```release-note
NONE
```

[MM-60837]: https://mattermost.atlassian.net/browse/MM-60837?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by devinbinnie — APPROVED** [review]
LGTM! Thanks @ZubairImtiaz3!

File: webapp/channels/src/components/user_settings/security/user_access_token_section/user_access_token_section.tsx [hunk 1]
```diff
@@ -264,11 +263,15 @@ export default class UserAccessTokenSection extends React.PureComponent<Props, S
             ),
             confirmMessage: () => (
                 <div className='alert alert-danger'>
-                    <FormattedMarkdownMessage
-                        id='user.settings.tokens.confirmDeleteMessage'
-                        defaultMessage='Any integrations using this token will no longer be able to access the Mattermost API. You cannot undo this action. \n \nAre you sure want to delete the **{description}** token?'
+                    <FormattedMessage
+                        id='user.settings.tokens.confirmDeleteMessageText'
+                        defaultMessage={
+                            'Any integrations using this token will no longer be able to access the Mattermost API. You cannot undo this action.{br}{br}Are you sure you want to delete the <b>{description}</b> token?'
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/user_settings/security/user_access_token_section/user_access_token_section.tsx:269]: lets break it down into two texts.

File: webapp/channels/src/components/user_settings/security/user_access_token_section/user_access_token_section.tsx [hunk 2]
```diff
@@ -264,11 +263,15 @@ export default class UserAccessTokenSection extends React.PureComponent<Props, S
             ),
             confirmMessage: () => (
                 <div className='alert alert-danger'>
-                    <FormattedMarkdownMessage
-                        id='user.settings.tokens.confirmDeleteMessage'
-                        defaultMessage='Any integrations using this token will no longer be able to access the Mattermost API. You cannot undo this action. \n \nAre you sure want to delete the **{description}** token?'
+                    <FormattedMessage
+                        id='user.settings.tokens.confirmDeleteMessageText'
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/user_settings/security/user_access_token_section/user_access_token_section.tsx:267]:
```suggestion
                        id='user.settings.tokens.confirmDelete'
```

File: webapp/channels/src/components/user_settings/security/user_access_token_section/user_access_token_section.tsx (hunk 1 above)
Comment (ZubairImtiaz3) [reply webapp/channels/src/components/user_settings/security/user_access_token_section/user_access_token_section.tsx:269]: Done.

**Review by M-ZubairAhmed — APPROVED** [review]
This looks good :shipit: Thank you

**Review by yasserfaraazkhan — APPROVED** [review]

**Pull request merged**